addons/mail/static/tests/owl/component/composer_tests.js

```diff
@@ -195,22 +201,34 @@ QUnit.skip('display partner mention suggestions on typing "@"', async function (
     document.querySelector(`.o_ComposerTextInput_textarea`)
         .dispatchEvent(new window.KeyboardEvent('keyup'));
     await afterNextRender();
-    assert.strictEqual(
-        document.querySelectorAll(`.tribute-container`).length,
-        1,
-        "should display the tribute mention suggestions on typing '@'"
+    assert.ok(
+        document.querySelector('.o_ComposerTextInput_mention_dropdown .dropdown-menu').classList.contains('show'),
+        "should display mention suggestions on typing '@'"
     );
 });
 
-// Test skipped until mentions manager is ready
-QUnit.skip('mention a partner', async function (assert) {
-    assert.expect(4);
+QUnit.test('mention a partner', async function (assert) {
+    assert.expect(2);
 
-    await this.start();
+    await this.start({
+        async mockRPC(route, args) {
+            if (args.method === 'get_mention_suggestions') {
+                const result = [[
+                    {
+                        id: 0,
```

Comment:
avoid using id 0 because it can lead to issue in the code when testing for existence. 2 would be fine (default value for odoo bot partner)

Reply:
Done